src/pages/projectView/projectMainBody/components/ProjectRewardPanel.tsx

```diff
@@ -70,10 +70,11 @@ export const ProjectRewardPanel = ({ reward }: Props) => {
                     variant='secondary'
                     size='sm'
                     px={2}
-                    onClick={() => {
+                    onClick={async () => {
                         updateReward({ id: toInt(reward.id), count: 1 })
-                        navigate(PathName.projectRewards)
+                        await navigate(PathName.projectRewards)
                         setMobileView(MobileViews.funding)
+                        setFundingFormState('step', 'contribution')
```

Comment:
I don't think await has any affect on the `navigate` function. I just verified it doesnot.

Reply:
It does - if you don't add that await on mobile it will not navigate in the sidebar to the funding view.